### ISSUE TITLE
Keyboard sensor cannot work only for pressing

### ISSUE THREAD

**Issue by vlad1777d** [UPBGE/blender#231]
Hello friends.

In this issue: https://github.com/UPBGE/blender/issues/196 I suggested to add the **button** to keyboard sensor, which will answer to **activating mode**: 
-pressed: keyboard sensor activates on releasing and pressing key;
-unpressed: keyboard sensor activates only on pressing the key;
Because people, who did not use Python, only a logic bricks, very often want sensor be activated only if key was pressed, not twice: when pressed and when released.

In IRC @panzergame told me that such behavior I can achieve with "Tap" button, and those issue was successfully closed.
This is not so, in this expample: [keyboard_issue.blend.zip](https://github.com/UPBGE/blender/files/496591/keyboard_issue.blend.zip) you can see, that after pressing "space" (with activated "Tap" button), Python controller **activates twice**:
![2016-09-27 23-42-48](https://cloud.githubusercontent.com/assets/12682937/18891138/9e967e50-850c-11e6-9a6d-fce0aa4dd9f0.png)

Maybe it's a **bug** with tap button, than it needs to be corrected. 
If it's **normal behavior**, than, I think, there is need to add "**activate mode**" button.

Because people, who don't know Python could not simply achieve such behavior: one pressing - one operation.
With other combinations of current buttons we could not reach needed behavior.

**Such behavior is needed when**:
\> We need to increase amount of something;
\> add some object to scene (otherwise, with a tab button 2 objects will be added to one place);
\> make purchase (increase one property, decrease second) (of course, we could divide amounts into two and get same result, but it's not good way);
\> one pressing of one key activates animation, second pressing of this key stops it;
\> move object to certain distance and etc.

With using Python this problem is solved easily, without it - sometimes very difficultly or impossible. 
I know, that without Python it's almost impossible to build a complex game, but I think, that we must take into account and those people, who make not difficult scenes "for themselves" and don't know Python.

Best regards,
Vladislav.

**Comment by youle31** [comment]
This is normal and we can't change that (@panzergame: I already talked to Moguri about that and he told me that it was the right behavior). Tap send 2 pulses: just activated + just released. If you want to do something only once in your python script, check sensor status with if cont.sensors["name"].status == bge.logic.KX_INPUT_JUST_ACTIVATED: do something. Thanks for report

**Comment by vlad1777d** [comment]
@youle31 , yes, right. 
But what about people, who don't use Python?

**Comment by panzergame** [comment]
@vlad1777d : All actuators are doing the if positive test.

**Comment by vlad1777d** [comment]
@panzergame I mean how people, who are making simple scene without python, can make activating their action only by they pressed key ?
(not by pressing and releasing)

**Comment by panzergame** [comment]
@vlad1777d : I really don't understand your issue. You mean loop the action when the key is pressed ?

**Comment by vlad1777d** [comment]
@panzergame , no, I mean next:
many times (I described in first message) there is need to activate actuator only one time - on pressing (or releasing) key.
>Key "space" was pressed - actuator "a" launched one time.

**UPBGE had such behavior** before you closed https://github.com/UPBGE/blender/issues/196 , and I'm sure, **such behavior**, as was present before closing bug, **is needed** (and I wrote it in that bugreport). 
It's needed for people , who doesn't know Python and\or makes simple scenes (other people could make checks of sensor's state in Python) (for reasons, that I described here in first message and I'm duplicating them below).

**Such behavior is needed when**:
\> We need to increase amount of something;
\> add some object to scene (otherwise, with a tab button 2 objects will be added to one place);
\> make purchase (increase one property, decrease second) (of course, we could divide amounts into two and get same result, but it's not good way);
\> one pressing of one key activates animation, second pressing of this key stops it;
\> move object to certain distance and etc.

**I wrote that bugreport because that behavior did not allowed activating scripts on key releasing** using keyboard sensor (allowed only using always sensor with checks through Python every frame).
But I also noticed, that that behavior is very useful (and, we can say, necessary) and there is need to save it. For example, by **adding "activating mode" button**.
